### ISSUE TITLE
Bugs Blocking Screen Reader Users

### ISSUE THREAD

**Issue by bbertucc** [1111philo/plato#39]
A screen reader user using VoiceOver for mac reported:

When I navigate VoiceOver actually behaves normally. When I type nothing is spoken. When I pressed the Send button nothing was spoken. When I switched to a different software programme immediately after I sent a message nothing was spoken. Nothing is spoken automatically in any scenario. If that is intended behaviour then okay.
Some other things I think must be fixed.
* When the objective pop-up thing is revealed VoiceOver fails to switch to it. Once it did but I was not able to make it set focus to it by opening it. I suggest to set focus to the beginning of that dialogue.
* Immediately before the text field there are two textual elements: new message from coach and your message. These seem to be erroneous. It seems that these are to be spoken automatically but not actual text.
* My messages and coach messages VoiceOver calls articles. That may or may not be correct but I do not know.
* There is an element that holds all of the articles and VoiceOver states the entire [foundations n] [title of lesson]. I think that is unnecessary and also repetitious to hear that so frequen Maybe something like chat log or conversation log will suffice.
* add a keyboard handler for the Command+Return shortcut to send a message. I must navigate to the Send button but Command+Return would work well because just Return inserts a new line which is good. Command+Return would put it inline with Slack.
